transactions/build.gradle

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+  compile project(':impl'), project(':xml')
+  compile group: 'javax.transaction', name: 'jta', version: '1.1'
+  testCompile project(path: ':core-spi-test')
+  compile group: 'org.codehaus.btm', name: 'btm', version: '2.1.4'
```

Comment:
Because BTM (and probably some other TM's too) need some glue code to work with non-JMS / non-JDBC XAResource's.